libmscore/note.cpp

```diff
@@ -3442,7 +3442,7 @@ Shape Note::shape() const
       Shape shape(r);
       for (NoteDot* dot : _dots)
             shape.add(symBbox(SymId::augmentationDot).translated(dot->pos()));
-      if (_accidental)
+      if (_accidental && _accidental->addToSkyline())
             shape.add(_accidental->bbox().translated(_accidental->pos()));
       for (auto e : _el) {
             if (e->autoplace() && e->visible()) {
```

Comment:
Minor nit, not your code of course, but this could now be addToSkyline() too if we wanted.